tools/create_triage_tickets.py

```diff
@@ -169,6 +199,8 @@ def main(arg):
              '{signature_type: {root_issue: message}}',
         default='./triage_resolving_filters.json',
     )
+    parser.add_argument("--mark-root", default=None, required=False, help="Location of the logs for marking as "
```

Comment:
not sure about the name, but also I don't have any other suggestion than ``mark-root``
@YuviGold wdyt?